### DISCUSSION TITLE
“docker compose -f docker-compose.yml up -d” this code seems to have been deprecated？

### DISCUSSION THREAD

**Discussion by 1119765208** [infiniflow/ragflow#3503] [Q&A]
the error message is as follows:
level=warning msg="The \"ELASTIC_PASSWORD\" variable is not set. Defaulting to a blank string."
level=warning msg="D:\\ragflow\\docker\\docker-compose-base.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"
level=warning msg="D:\\ragflow\\docker\\docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"

does anyone know what this is?thanks.

**Comment by KevinHuSh** [comment]
Docker version 26.1.2
